infrastructure/environments/dplplat01/configuration/mariadb-servers/values.yaml

```diff
@@ -0,0 +1,11 @@
+namespace: mariadb-servers
```

Comment:
You don't need to include the namespace at all, as that is defined when installing the helm chart. Just leave out the field in the resources

Reply:
hvad sker der hvis ikke der står noget namespace of man glemmer at sætte nogen af `namespace` flagene når man køre `helm install`? Lander serveren så ikke i `default` namespacet?